src/main/java/seedu/address/logic/commands/ReadMoreCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AddressBookModel;
+import seedu.address.model.DiagnosisModel;
+import seedu.address.model.ScheduleModel;
+import seedu.address.model.DrugSearchUtility;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Displays extended pharmacological information about a given drug from search results.
+ */
+
+public class ReadMoreCommand extends Command {
+
+    public static final String COMMAND_WORD = "readmore";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "Displays extended pharmacological information "
+            + "about any matching drug from the most recent search of the drug database.\n"
+            + "Parameters: INDEX (must be from the list of search results)...\n"
+            + "Example: "
+            + COMMAND_WORD
+            + " readmore"
+            + " 3\n";
+
+    private final int searchIndex;
+
+    public ReadMoreCommand(int searchIndex) {
```

Comment:
what index is this referencing?

Reply:
The indices are displayed as part of the search results. In order to read more about any particular result, the user may enter "readmore [index]". The DrugSearchUtility class has a static member which stores the data corresponding to particular indices.

Reply:
Actually it was already there, but I had described the command as "moreinfo" instead of "readmore". Closing this PR now and adding a new one, look at the change to DrugSearchUtility.java.